misc/requirements_wheel.txt

```diff
@@ -1,10 +1,10 @@
 # numpy pinning for ABI forward-compatibility
 numpy==1.16.5 ; python_version < "3.8" and platform_machine !='aarch64'
 numpy==1.17.* ; python_version == "3.8" and platform_machine !='aarch64'
-numpy==1.19.4 ; python_version >= "3.9" and platform_machine !='aarch64'
+numpy==1.19.4 ; python_version == "3.9" and platform_machine !='aarch64'
 
 # NOTE: oldest-supported-numpy (1.19.2) had forward ABI compat problems
-numpy==1.20.* ; platform_machine=='aarch64'
+numpy==1.21.*
```

Comment:
I think the change should be:
```
numpy==1.20.* ; python_version < "3.10" and platform_machine=='aarch64'
numpy == 1.21.* ; python_version >= "3.10"
```

- keeps the 1.20 pin for python 3.9 + aarch64
- pin 1.21 for >=3.10 unless/until we have to manually adjust for another release